src/pages/en/guides/migrate-to-astro/from-nuxtjs.mdx

```diff
@@ -2,68 +2,682 @@
-
-- Astro has [an official integration for using Vue components](/en/guides/integrations-guide/vue/) and supports [installing NPM packages](/en/guides/imports/#npm-packages), including several for Vue. You will be able to write Vue UI components, and may be able to keep some or all of your existing components and dependencies.
-
-- Astro and Nuxt both allow you to use a [headless CMS, APIs or Markdown files for data](/en/guides/data-fetching/). You can continue to use your preferred [content authoring](/en/guides/content/) system, and will be able to keep your existing content.
+- Astro projects can also be SSG or [SSR with page level prerendering](/en/guides/server-side-rendering/#prerendering).
+- Astro uses file-based routing, and [allows specially named pages to create dynamic routes](/en/core-concepts/routing/#dynamic-routes).
+- Astro is [component-based](/en/core-concepts/components/), and your markup structure will be similar before and after your migration.
+- Astro has [an official integration for using Vue components](/en/guides/integrations-guide/vue/).
+- Astro has support for [installing NPM packages](/en/guides/imports/#npm-packages), including Vue libraries. You may be able to keep some or all of your existing Vue components and dependencies.
 
 ## Key Differences between Nuxt and Astro
 
 When you rebuild your Nuxt site in Astro, you will notice some important differences:
 
-- [MPA vs SPA](/en/concepts/mpa-vs-spa/): Nuxt is a React-based SPA (single-page application). Astro sites are multi-page apps built using [`.astro` components](/en/core-concepts/astro-components/), but can also support [React, Preact, Vue.js, Svelte, SolidJS, AlpineJS, Lit](/en/core-concepts/framework-components/) and raw HTML templating.
+- [MPA vs SPA](/en/concepts/mpa-vs-spa/): Nuxt is a Vue-based SPA (single-page application). Astro sites are multi-page apps built using `.astro` components, but can also support React, Preact, Vue.js, Svelte, SolidJS, AlpineJS, Lit and raw HTML templating.
 
 - [Page Routing](/en/core-concepts/astro-pages/#file-based-routing): Nuxt uses `vue-router` for SPA routing, and `vue-meta` for managing `<head>`. In Astro, you will create separate HTML page routes and control your page `<head>` directly, or in a layout component.
 
-- [Content-focus](/en/concepts/why-astro/): Astro was designed to excel at making content-focused websites. An existing Nuxt app might be built for high client-side interactivity and may include items that are difficult to replicate in Astro, such as dashboards.
+- [Content-focus](/en/concepts/why-astro/): Astro was designed primarily for content-focused sites. While Nuxt has the capability to do content-focused sites through `@nuxt/content`, it's primarily geared towards interactive websites. This differs from Astro, which has built-in capabilities for content page generation, but may be difficult to replicate interactive components such as dashboards within.
```

Comment:
I wonder if we're being too conservative about Astro's capabilities with the "interactive components" wording - Astro works great with interactive components, but it's shared state on the client (across multiple islands and across pages) that makes things more difficult.

```suggestion
- [Content-focus](/en/concepts/why-astro/): Astro was designed to excel at making content-focused sites. While Nuxt has the capability to do content-focused sites through `@nuxt/content`, an existing Nuxt app may be built for high client-side interactivity. Astro has built-in capabilities for working with your content, such as page generation, but it may be difficult to replicate items with complex shared state, such as dashboards.
```